### PR TITLE
New seg api

### PR DESCRIPTION
**Pull request by ronzim**
BREAKING CHANGE: redefine getActiveLabelmapBuffer
feat: add loadMaskSlice
feat: add toggleVisibility
feat: set/get labelColor
feat: add config in initSegmentationModule

### CODE REVIEW

File: imaging/tools/segmentation.js [hunk 1]
```diff
@@ -6,18 +6,13 @@
...
 // internal libraries
 import { setToolActive, setToolDisabled } from "./main";
 
-/**
- * THIS MODULE IS A WIP
- */
-
 // DEV
 window.segModule = segModule;
 window.cornerstone = cornerstone;
```
Comment (daron1337) [inline imaging/tools/segmentation.js:18]: Remove these lines (cornerstone is already exported in index.js btw)

**Review by daron1337 — CHANGES_REQUESTED** [review]
- Change version in package.json and README
- Update compiled larvitar.js in examples

**Comment by ronzim** [comment]
> * Change version in package.json and README
> * Update compiled larvitar.js in examples

examples are not affected by updates

File: imaging/tools/segmentation.js [hunk 2]
```diff
@@ -191,6 +190,21 @@ export function setActiveLabelmap(labelId, elementId) {
...
+ * Get active labelmap for target element
+ * @param {String} elementId
+ * @returns {Object} The active labelmap object that contains the buffer
+ */
+export function getActiveLabelmapBuffer(elementId) {
+  if (!elementId) {
+    console.error(
+      "getActivelabelmapBuffer now needs elementId as param and returns the buffer parent object"
+    );
+  }
```
Comment (daron1337) [inline imaging/tools/segmentation.js:236]: In order to be consistent with other functions that uses elementId in larvitar, use the isElement function inside

**Review by daron1337 — CHANGES_REQUESTED** [review]
- Fix the function call using isElement
- Document the breaking change in the Migration.md file

File: imaging/tools/segmentation.js (hunk 2 above)
Comment (daron1337) [reply imaging/tools/segmentation.js:236]: `import { isElement } from "../imageUtils";`

`let element = isElement(elementId)
    ? elementId
    : document.getElementById(elementId);
  if (!element) {
    console.error("invalid html element: " + elementId);
    return;
  }`

**Review by daron1337 — APPROVED** [review]

**Pull request merged**